### PR TITLE
[DOC-9553] Correct the note about updating the CA certificate, document how to roll out a CA certificate update gradually

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-9553] Correct the note about updating the CA certificate, document how to roll out a CA certificate update gradually

### CODE REVIEW

**Review by cjireland — COMMENTED** [review]
Line 242 is not true if multiple CA certs are included before (line 245).

**Comment by mdlinville** [comment]
@cjireland approved the current state of the PR out of band.

**Review by shannonbradshaw — APPROVED** [review]
LGTM

File: src/current/cockroachcloud/client-certs-dedicated.md [hunk 1]
```diff
@@ -239,12 +239,10 @@ resource "cockroach_client_ca_cert" "yourclustername" {
...
 
 {{site.data.alerts.callout_danger}}
-Clients must be provisioned with client certificates signed by the new CA prior to the update, or their new connections will be blocked.
-
-This operation also interrupts existing database connections. End users should be informed of a potential service interruption.
+Clients must be provisioned with client certificates signed by the cluster's CA prior to the update, or their new connections will be blocked.
 {{site.data.alerts.end}}
 
-This section shows how to replace the CA certificate used by your cluster for certificate-based client authentication.
+This section shows how to replace the CA certificate used by your cluster for certificate-based client authentication. To roll out a new CA certificate gradually instead of following this procedure directly, CockroachDB supports the ability to include multiple CA certificates for a cluster by concatenating them in PEM format. This allows clients to connect as long as the client certificate is signed by either the old CA certificate or the new one. PEM format requires a blank line in between certificates.
```
Comment (BabuSrithar) [inline src/current/cockroachcloud/client-certs-dedicated.md:245]: @mdlinville , We have coniremd that the operation does not interrupt existing connections, but we have not confirmed if the Multiple CA functions as expected. I would remove this till we have this confirmed. Sorry that I got to this after the merge.

Comment (bdarnell) [reply src/current/cockroachcloud/client-certs-dedicated.md:245]: Once we've confirmed this, we should pluralize "CA certificates" at least in the update section and possibly throughout the whole page. No one should update directly from one single certificate to another single certificate (except in the case of a compromised key); the way to update your certificates is to go through a two-stage process with a period when both CAs are accepted.

**Pull request merged**